lib/src/base-styles/_nx-colors.scss

```diff
@@ -92,6 +92,21 @@
   --nx-color-counter: var(--nx-swatch-grey-05);
   --nx-color-counter-active-text: var(--nx-swatch-white);
 
+  //NxProgressBar
+  --nx-color-progress-bar-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-bar-progress: var(--nx-swatch-teal-60);
+  --nx-color-progress-bar-text-success: var(--nx-swatch-green-25);
+  --nx-color-progress-bar-icon-success: var(--nx-color-progress-bar-text-success);
```

Comment:
Eliminate this variable and use `--nx-color-validation-valid-icon` directly in NxProgressBar

Reply:
> Eliminate this variable and use `--nx-color-validation-valid-icon` directly in NxProgressBar

What do you think of setting the value of `--nx-color-progress-bar-icon-success:` to `--nx-color-validation-valid-icon`

```
--nx-color-progress-bar-icon-success: var(--nx-color-validation-valid-icon);
```

Using `--nx-color-validation-valid-icon` on NxProgressBar's success icon directly could work. But  "valid" and "success" are similar but not the same thing. To me, `--nx-color-progress-bar-icon-success` is a more accurate variable name.

Reply:
> > Eliminate this variable and use `--nx-color-validation-valid-icon` directly in NxProgressBar
> 
> What do you think of setting the value of `--nx-color-progress-bar-icon-success` to `--nx-color-validation-valid-icon`?
> 
> ```
> --nx-color-progress-bar-icon-success: var(--nx-color-validation-valid-icon);
> ```
> 
> Using `--nx-color-validation-valid-icon` on NxProgressBar's success icon directly could work. But "valid" and "success" are similar but not the same thing. To me, `--nx-color-progress-bar-icon-success` is a more accurate variable name.

Per the WW conversation, it was decided that we should use new variables over --nx-color-validation-invalid/valid-icon. For more details, see the comment below.
https://github.com/sonatype/sonatype-react-shared-components/pull/1049#discussion_r1158733157

I think this conversation is resolved so I'm closing it.